src/legacy/core_plugins/vis_type_table/index.ts

```diff
@@ -24,7 +24,7 @@ import { LegacyPluginApi, LegacyPluginInitializer } from '../../../../src/legacy
 
 const tableVisPluginInitializer: LegacyPluginInitializer = ({ Plugin }: LegacyPluginApi) =>
   new Plugin({
-    id: 'table_vis',
+    id: 'vis_type_table',
```

Comment:
let's wait on renaming this and the corresponding `package.json` id. Per [Tim's note in the TSVB PR](https://github.com/elastic/kibana/pull/39169/files#r297685986), this will break config.yml settings for enabling/disabling the plugin so it's only something we can merge into `master` for 8.0.

I'm thinking it might be best to go through and make these breaking changes further into the migration process. I'll create an issue to remind us.

I've also realized that we already changed this for markdown -- would you be willing to revert this in your markdown PR #41760?

Reply:
Issue opened for changing IDs: #41931

Reply:
ID was reverted